EdgeCraftRAG/tests/test_compose_on_arc.sh

```diff
@@ -30,26 +30,26 @@ HF_ENDPOINT=https://hf-mirror.com
 
 
 function build_docker_images() {
-    opea_branch=${opea_branch:-"main"}
     cd $WORKPATH/docker_image_build
-    git clone --depth 1 --branch ${opea_branch} https://github.com/opea-project/GenAIComps.git
-    pushd GenAIComps
-    echo "GenAIComps test commit is $(git rev-parse HEAD)"
-    docker build --no-cache -t ${REGISTRY}/comps-base:${TAG} --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
-    popd && sleep 1s
-
     echo "Build all the images with --no-cache, check docker_image_build.log for details..."
-    service_list="edgecraftrag edgecraftrag-server edgecraftrag-ui"
     docker compose -f build.yaml build --no-cache > ${LOG_PATH}/docker_image_build.log
 
     docker images && sleep 1s
 }
 
 function start_services() {
+    export MODEL_PATH=${MODEL_PATH}
+    export DOC_PATH=${DOC_PATH}
     export UI_UPLOAD_PATH=${UI_UPLOAD_PATH}
+    export HOST_IP=${HOST_IP}
+    export LLM_MODEL=${LLM_MODEL}
+    export HF_ENDPOINT=${HF_ENDPOINT}
+    export vLLM_ENDPOINT=${vLLM_ENDPOINT}
+    export HUGGINGFACEHUB_API_TOKEN=${HUGGINGFACEHUB_API_TOKEN}
+    export no_proxy="localhost, 127.0.0.1, 192.168.1.1"
 
     cd $WORKPATH/docker_compose/intel/gpu/arc
-    source set_env.sh
```

Comment:
We prefer to use `set_env.sh` in both test and readme, so when the env updated we only need to update one place.